### PR TITLE
3407: add associatedJudge to case entity and case detail header

### PR DESCRIPTION
**Pull request by rachaelparris**
We need to clear dev dynamo when this is merged due to a change in the case entity validation.
Screen Shot 2019-11-05 at 8 45 38 AM

### CODE REVIEW

File: web-client/src/views/CaseDetailHeader.jsx [hunk 1]
```diff
@@ -45,6 +45,23 @@ export const CaseDetailHeader = connect(
...
+                      aria-label="associated judge"
+                      className="margin-left-1 usa-tag"
+                    >
+                      <span aria-hidden="true">
+                        <FontAwesomeIcon
+                          className="margin-right-05"
+                          icon="gavel"
+                          size="1x"
+                        />
+                        {formattedCaseDetail.associatedJudge}
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailHeader.jsx]: Is there a reason to provide "associated judge" on the outer span's aria-label, but hide the name of the *actual* judge?  Perhaps the name should appear outside the hidden fa-icon?

**Review by kkoskelin — APPROVED** [review]
Looks okay! just a question about the aria attribute usage on the associated judge.

File: web-client/src/views/CaseDetailHeader.jsx (hunk 1 above)
Comment (matthopson) [reply web-client/src/views/CaseDetailHeader.jsx]: @kkoskelin What's your take on:

>Don't use aria-label or aria-labelledby on a span or div unless its given a role. When aria-label or aria-labelledby are on interactive roles (such as a link or button) or an img role, they override the contents of the div or span. Other roles besides Landmarks (discussed above) are ignored.

From: https://www.w3.org/TR/using-aria/

I feel like I've seen us use `aria-label` on span tags. Are these being ignored?

Comment (kkoskelin) [reply web-client/src/views/CaseDetailHeader.jsx]: Good question -- I guess I wasn't completely aware of this convention. I have indeed added `aria-label` to elements to simply provide "alternative labels" to be provided to assistive tech, rather than the text found within them.  To my knowledge they are NOT being ignored by MacOS Voiceover + Safari, but are being read aloud.

Comment (waldoj) [reply web-client/src/views/CaseDetailHeader.jsx]: Whoa — I was also not aware of this convention.

Comment (matthopson) [reply web-client/src/views/CaseDetailHeader.jsx]: I personally only test with Mac's VoiceOver utility - which may very well ignore some standard conventions and read everything.

Here's another article that's specific to JAWS (windows screen reader).
https://stackoverflow.com/questions/42681485/div-aria-label-not-being-read-by-jaws

I see similar questions surfacing around NVDA (another popular windows reader).
https://www.nvaccess.org/

Comment (matthopson) [reply web-client/src/views/CaseDetailHeader.jsx]: @waldoj @kkoskelin 
There's also this:
https://webaim.org/projects/screenreadersurvey7/

TLDR;
- Blind people rely heavily on screen readers.
- The most common tech stack (overwhelmingly so) is Windows + Firefox + JAWS or NVDA.

Comment (waldoj) [reply web-client/src/views/CaseDetailHeader.jsx]: [Survey #8 is out now, too](https://webaim.org/projects/screenreadersurvey8/)! WebAIM's research is great stuff, really essential.
